admin/tests/integration/components/certification-info-competences-test.js

```diff
@@ -45,7 +46,7 @@ module('Integration | Component | certification-info-competences', function(hook
     this.set('competences', [{ index:'1.1', score:'30', level:'3' }, { index:'2.1', score:'30', level:'3' }, { index:'5.2', score:'30', level:'3' }]);
 
     // When
-    await render(hbs`{{certification-info-competences competences=competences edition=true}}`);
+    await render(hbs`<CertificationInfoCompetences @competences={{competences}} @edition={{true}} />`);
```

Comment:
`competences` n'est pas plutôt rattaché au `this` ?

```suggestion
    await render(hbs`<CertificationInfoCompetences @competences={{this.competences}} @edition={{true}} />`);
```

Reply:
Done